src/main/java/com/tabnine/inline/CompletionPreviewUtils.kt

```diff
@@ -5,3 +5,10 @@ import com.tabnine.prediction.TabNineCompletion
 fun hadSuffix(currentCompletion: TabNineCompletion): Boolean {
     return currentCompletion.oldSuffix?.trim()?.isNotEmpty() ?: false
 }
+
+fun isSingleLine(currentCompletion: TabNineCompletion): Boolean {
+    return !currentCompletion.getSuffix().trim()?.contains("\n") ?: true
+}
+fun shouldRemoveSuffixIfSingleLine(currentCompletion: TabNineCompletion): Boolean {
```

Comment:
Remove "IfIsSingleLine" from the name - it's an internal detail to this function and irrelevant to the caller